clients/clearly-defined/src/main/kotlin/Errors.kt

```diff
@@ -38,6 +38,10 @@ data class Error(
 
 @Serializable
 data class InnerError(
+    val ok: Int,
```

Comment:
No, this is unfortunately not really documented. The closest you get probably is https://github.com/clearlydefined/service/blob/104d4d8718678c8804e2124da7fbb891c0c1e57c/app.js#L262.

Reply:
Would it make sense to add default values for all (new) properties, to avoid running into de-serialization issues? ...if it's undocumented, I'm not confident relying on these to always be present.

Reply:
I went for a different solution that ignores unknown fields just for the errors.